src/openrct2/ui/UiContext.h

```diff
@@ -29,6 +29,8 @@ namespace OpenRCT2
         interface IRainDrawer;
         using DrawRainFunc = void (*)(
             OpenRCT2::Drawing::IRainDrawer* rainDrawer, int32_t left, int32_t top, int32_t width, int32_t height);
+        using DrawSnowFunc = void (*)(
```

Comment:
You only need one prototype for the weather drawing. Rename DrawRainFunc to DrawWeatherFunc